src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -359,6 +359,12 @@ private void ConfigureEnvironment()
             {
                 case CreationResultStatus.Success:
                     Reporter.Output.WriteLine(string.Format(LocalizableStrings.CreateSuccessful, resultTemplateName));
+
```

Comment:
Nit: This display code appears identically in 2 places, here, and below in ShowTemplateHelp(). It's pretty trivial, but could become more complex at some point. Just concerned with New3Command.vs getting more polluted.